polls/templates/polls/base.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
```

Comment:
You need a `{% block body %}{% endblock %}` in here otherwise your other templates won't know where to inject your HTML.

Reply:
fixed it and pushed